blockly/build.gradle

```diff
@@ -7,7 +7,7 @@ repositories {
 }
 
 dependencies {
-  implementation project(':dungeon')
+  implementation project(':dungeon'), 'com.github.javaparser:javaparser-symbol-solver-core:3.26.4'
```

Comment:
@cagix Ist das okay, das ich eine neue Abhängigkeit mit reinbringe? Ich verwende Javaparser für das Auslesen der JavaDocs für den Language Server in VS-Code

Reply:
Ich hätte 3 andere Wege (die imo schlechter wären):
1. Annotations
Hier wäre das "Problem" das ich für jede Methode 2x die Doku im Sourcecode hätte.
2. Eigner Annotation Processor oder Doclet API
So könnte ich zur Compilezeitpunkt die JavaDoc auslesen und irgendwie speichern.
3. JavaDoc zu HTML bauen

Das Problem bei Punkt 2 & 3 ist der Zeitaufwand, ich müsste einmal die Technik versehen und dann noch die Generierte JavaDoc parsen um sie wieder einzulesen. Punkt 1 wäre eigentlich nur hässlich im Code und etwas mehr Aufwändig wenn man den Code erweitert.

Reply:
hmm. dann mach.

ich würde es gern irgendwo dokumentieren. hast du eine idee, wo das geschickt passieren könnte? hier im pr findet man das nie wieder.

Reply:
> hier im pr findet man das nie wieder.

hah. wenn diese abhängigkeit in einem eigenen pr reinkommen würde, könntest du das in der ersten message beschreiben. das geht dann als commit-message in die history rein. fertig. 

den commit findet man wieder, und damit dann auch die "doku".

@Flamtky passt das? du müsstest freilich dann das einfügen der dependency hier nochmal rauslösen und einzeln machen (und danach hier rebasen mit `--onto`).